source/Government.cpp

```diff
@@ -169,31 +169,17 @@ const Color &Government::GetColor() const
 
 
 // Get the government's tribute-related hails
-const Phrase *Government::GetTributeHail(int condition)
+const Phrase *Government::GetTributeHail(int condition) const
 {
-	switch(condition)
-	{
-		case 1:
-		{
-			return tributePresentHail;
-		}
-		case 2:
-		{
-			return tributeIgnoredHail;
-		}
-		case 3:
-		{
-			return tributeCombatHail;
-		}
-		case 4:
-		{
-			return tributeInProgressHail;
-		}
-		default:
-		{
-			return tributeSurrenderedHail;
-		}
-	}
+	static const vector<const Phrase *> phrases = {
+	tributePresentHail,
+	tributeIgnoredHail,
+	tributeInProgressHail,
+	tributeCombatHail,
+	tributeSurrenderedHail
+	};
+	
+	return phrases[condition];
```

Comment:
could you make this `return phrases[condition]->Get()`? then you change it to a std::string return value, and position it in the `.h` and `.cpp` right next to Government::GetHail() for supreme logical consistency :1st_place_medal: 

You may also want bounds checking on the input, though in your current code there's no variable inputs to this functions caller so you are pretty secure in knowing you are in-bounds.